indico/migrations/versions/20171101_1149_566d5de4e0e5_add_is_public_column_to_contribution_types.py

```diff
@@ -0,0 +1,29 @@
+"""Add is_private column to contribution types table
+
+Revision ID: 566d5de4e0e5
+Revises: 1d512a9ebb30
+Create Date: 2017-11-01 11:49:47.532339
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '566d5de4e0e5'
+down_revision = '1d512a9ebb30'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        'contribution_types',
+        sa.Column('is_private', sa.Boolean(), nullable=False, server_default=sa.true()),
```

Comment:
`server_default='true'` would be enough, but nothing wrong with letting SA generate that string for you